doc/release-notes/10.2.md

```diff
@@ -0,0 +1,26 @@
+# Release notes for Gluster 10.2
+
+This is a bugfix and improvement release. The release notes for [10.0](10.0.md) and [10.1](10.1.md) contain a listing of all the new features that were added and bugs fixed in the GlusterFS 10 stable release.
+
+**NOTE:**
+- Next minor release tentative date: Week of 15th May, 2022
+- Users are highly encouraged to upgrade to newer releases of GlusterFS.
+
+## Important fixes in this release
+- Optimize server functionality by enhancing server_process_event_upcall code path during the handling of upcall event
+- Fix all bricks not starting issue on node reboot when brick count is high(>750)
+- Fix stale posix locks that appear after client disconnection
+
+## Builds are available at
```

Comment:
Rephrase "Builds will be available at"

Reply:
This should be fine as, this phrase serves true, though-out the time builds get available at the link mentioned

Reply:
okay.